packages/appstore/src/components/appstore-banner/svg-migration-banner.tsx

```diff
@@ -66,7 +67,11 @@ const SVGMigrationBanner = observer(() => {
                     <Localize i18n_default_text='Upgrade now' />
                 </Button>
             </div>
-            <AppstoreBannerImage image={image} className='appstore-banner__image' />
+            {is_dark_mode_on ? (
+                <AppstoreBannerImage image={image_dark_mode} className='appstore-banner__image' />
+            ) : (
+                <AppstoreBannerImage image={image} className='appstore-banner__image' />
+            )}
```

Comment:
<AppstoreBannerImage image={`is_dark_mode_on? image_dark_mode:image`} className='appstore-banner__image' />
will this work?

Reply:
yess